lib/bolt/cli.rb

```diff
@@ -183,6 +189,10 @@ def validate(options)
         raise Bolt::CLIError,
               "Option '--noop' may only be specified when running a task"
       end
+
+      if options[:subcommand] == 'apply' && (options[:object] && options[:code])
```

Comment:
Parens for grouping are unnecessary. But I guess they kind of make sense for what's being tested.

Reply:
Yeah I was sort of trying to logically divide this into "this validation step applies to the apply command" vs. "this is the case we're actually validating"